README.md

```diff
@@ -3,6 +3,15 @@
 BYU-DML machine learning algorithms or primitives created for DARPA's D3M project.
 These primitives are wrapped to fit within the D3M ecosystem.
 
+## How to update
+0. Update the forked d3m repo with the newest changes, and in this repo's home directory run the command `git pull --recurse-submodules`.  This will pull down the forked repo's changes. Then build, launch, and enter that container.
```

Comment:
We do not fork off the d3m repo. Do you mean the primtives_repo? Do you think a link would be good?

Reply:
That's a good point.  I meant the primitives repo and a link would be helpful!